plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7616,6 +7617,59 @@ public void testSelectFromPrestoViewReferencingHiveTableWithTimestamps()
         assertThat(query(nanosSessions, "SELECT ts FROM hive_timestamp_nanos.tpch." + prestoViewNameNanos)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123000000'");
     }
 
+    @Test(dataProvider = "legalUseColumnNamesProvider")
+    public void testUseColumnNames(HiveStorageFormat format, boolean formatUseColumnNames)
+    {
+        String lcFormat = format.name().toLowerCase(Locale.ROOT);
+        Session.SessionBuilder builder = Session.builder(getQueryRunner().getDefaultSession())
+                .setIdentity(Identity.forUser("hive")
+                        .withRole("hive", new SelectedRole(ROLE, Optional.of("admin")))
+                        .build());
+        if (format == HiveStorageFormat.ORC || format == HiveStorageFormat.PARQUET) {
+            builder.setCatalogSessionProperty(catalog, lcFormat + "_use_column_names", String.valueOf(formatUseColumnNames));
+        }
+        Session admin = builder.build();
+        String tableName = format("test_renames_%s_%s_%s", lcFormat, formatUseColumnNames, randomTableSuffix());
+        assertUpdate(admin, format("CREATE TABLE %s (id BIGINT, old_name VARCHAR, age INT, state VARCHAR) WITH (format = '%s', partitioned_by = ARRAY['state'])", tableName, format));
+        assertUpdate(admin, format("INSERT INTO %s VALUES(111, 'Katy', 57, 'CA')", tableName), 1);
+        assertQuery(admin, "SELECT * FROM " + tableName, "VALUES(111, 'Katy', 57, 'CA')");
+
+        assertUpdate(admin, format("ALTER TABLE %s RENAME COLUMN old_name TO new_name", tableName));
+
+        boolean doesntSeeOldData = formatUseColumnNames || NAMED_COLUMN_ONLY_FORMATS.contains(format);
```

Comment:
It seems clearer to invert this, since a boolean with "not" creates a double negative: `canSeeOldData`

Reply:
Inverted.